docs/upgrade_to_tails_4.rst

```diff
@@ -0,0 +1,59 @@
+Upgrading workstations to Tails 4
+---------------------------------
+
+.. important::
+
+   Before upgrading your *Admin Workstation* and your *Journalist Workstation*
+   to Tails 4, you must first ensure that the version of the SecureDrop code on
+   the workstation (which is used for administrative tasks and for configuring
+   the Tails desktop) is at 1.1.0 or a later version.
```

Comment:
This would be a good opportunity to switch to `|version|` for all references to 1.1.0 in this chapter, and to instruct admins to simply ensure that they're using the latest version. (1.1.0 is most likely fine, but there's no particular reason to keep carrying forward a reference to such an old release for this upgrade procedure.)

Reply:
removed older upgrade guides in 60130e3d0

Reply:
Of course! fixed in 60130e3d0